tests/unit_tests/steps/test_archive_objects.py

```diff
@@ -19,81 +18,82 @@
 
 
 class TestArchiveObjects:
-    '''Test the achive step.
-    '''
+    """
+    Tests the archive step.
+    """
+    def test_for_exes(self, stub_tool_box: ToolBox,
+                      fs, fake_process: FakeProcess) -> None:
```

Comment:
Not your fault of course - but that is a HORRIBLE name. Took me forever to find out what on earth `fs` might even be. Luckily there was one test that typed it :)

Reply:
Thanks for the update !